tests/components/tuya/__init__.py

```diff
@@ -26,7 +26,7 @@
     "clkg_nhyj64w2",  # https://github.com/home-assistant/core/issues/136055
     "clkg_wltqkykhni0papzj",  # https://github.com/home-assistant/core/issues/151635
     "co2bj_yakol79dibtswovc",  # https://github.com/home-assistant/core/issues/151784
-    "co2bj_yrr3eiyiacm31ski",  # https://github.com/home-assistant/core/issues/133173
+    "co2bj_yrr3eiyiacm31ski",  # https://github.com/orgs/home-assistant/discussions/842
```

Comment:
I have updated the fixture as it is more recent and contains the pm10 status which the original fixture didn't have